app/javascript/dashboard/routes/dashboard/conversation/ContactPanel.vue

```diff
@@ -218,8 +218,6 @@ export default {
             >
               <CustomAttributes
                 attribute-type="contact_attribute"
-                attribute-class="conversation--attribute"
```

Comment:
Why did we remove these?

Reply:
@scmmishra When I moved the styles to inline in both components, the prop became unnecessary, which is why it was removed. Did you face any issues during testing?